### PR TITLE
get only transmission files in zcat

### PR DESCRIPTION
**Pull request by londumas**
get only transmission files in zcat, send with something like this:
```
import picca.utils
zcat = '/project/projectdirs/desi/mocks/lya_forest/london/v1.1/quick-0.0/zcat.fits'
indir = '/project/projectdirs/desi/mocks/lya_forest/london/v1.1/*/*/transmission-16-*.fits'
outdir = '/global/cscratch1/sd/hdumasde/igmhub/picca/CoLoRe_mocks/v2.1.1/Test_convert_transmission_2/'
picca.utils.desi_convert_transmission_to_delta_files(zcat=zcat,indir=indir,outdir=outdir,nspec=10000)
```

### CODE REVIEW

**Comment by ngbusca** [comment]
Thanks @londumas ! does it work on both london and saclay mocks?

**Comment by londumas** [comment]
It works only on London mocks (Saclay doesn't have EXTNAME), either we wait 1 or 2 days for the latest Saclay mocks or I change it so that it reads both now. What do you prefer?

**Comment by ngbusca** [comment]
It would be great to have that now so the analysis will be ready for the meeting! But if the code gets too complicated, then don't worry about it.

File: py/picca/utils.py [hunk 1]
```diff
@@ -252,6 +278,8 @@ def desi_convert_transmission_to_delta_files(indir,outdir,lObs_min=3600.,lObs_ma
             deltas[pixnum].append(delta(thid[i],ra[i],dec[i],z[i],thid[i],thid[i],thid[i],cll,civ,None,cfl,1,None,None,None,None,None,None))
         if not nspec is None and sp.sum([ len(deltas[p]) for p in list(deltas.keys())])>=nspec: break
 
+    print('\n')
```
Comment (ngbusca) [inline py/picca/utils.py]: this will print two empty lines, is that what you want?

Comment (londumas) [reply py/picca/utils.py]: I just want to separate the two `sys.stderr.write`. What is the best way?

Comment (ngbusca) [reply py/picca/utils.py]: It's ok, eventually we will use one single function for screen output, probably a flushed `print()`

**Comment by londumas** [comment]
@ngbusca, ready to be merged.
It works for both London and Saclay mocks.
It is at the cost of not beautiful code. As soon as we have the two newest versions,
I will make a simple PR using EXTNAME.

**Pull request merged**